sdks/cpp/README.md

```diff
@@ -115,34 +115,49 @@ For more information, you can also read the [SDK Overview](../), check out [sdk.
 When running on Agones, the above functions should only fail under exceptional circumstances, so please 
 file a bug if it occurs.
 
-## Dynamic and Static Libraries
-
-In the [releases](https://github.com/googlecloudplatform/agones/releases) folder
-you will find two archives for download that contain both static and dynamic libraries for building your
-game server on Linux:
-
-- `argonsdk-$(VERSION)-dev-linux-arch_64.tar.gz`: This includes all the 
-headers as well as dynamic and static libraries that are needed for development and runtime.
-- `argonsdk-$(VERSION)-runtime-linux-arch_64.tar.gz`: This includes just the dynamic libraries that 
-are needed to run a binary compiled against the SDK and its dependencies.
-
-### Building the Libraries
+## Building From Source
+To build Agones C++ SDK from sources you need to set up your development environment and install following tools:
+- CMake (https://cmake.org/download/)
+- OpenSSL (https://github.com/openssl/openssl)
```

Comment:
Why does the user need to install OpenSSL? We actually don't need SSL termination in our SDK, so if we can remove this, that would be ideal. In the previous incarnation, we just built out grpc++_unsecure

Reply:
If I recall correctly, grpc doesn't offer much in terms of customization and it isn't technically possible to compile it without openssl.

Reply:
If I'm reading the previous PR correctly, it looks you can compile it without SSL. But maybe I'm reading it wrong? 🤷‍♂️

Reply:
Which previous PR? 
But if you look at grpc cmake https://github.com/grpc/grpc/blob/master/CMakeLists.txt#L141
and https://github.com/grpc/grpc/blob/master/cmake/ssl.cmake
there isn't really any way to turn it off.
GRPC cmake (I believe) is generated, and really hard to follow. 
It is probably possible to build specifically the `grpc_unsecure` target, and set `gRPC_SSL_PROVIDER`  to `module`. 

We really would need an automated cmake build for this repo though. A lot of surprises will pop up since its currently untested,

Reply:
This previous PR: https://github.com/GoogleCloudPlatform/agones/pull/421

> It is probably possible to build specifically the grpc_unsecure target, and set gRPC_SSL_PROVIDER to module

Can we do that :) ?

> We really would need an automated cmake build for this repo though. A lot of surprises will pop up since its currently untested,

Yes - agreed. We should integrate this PR into the cloudbuild system. Next steps should be to make some tests, and also setup CI with windows (AppVeyor?) and OSX (Travis I think does this?) as well.

Reply:
When you add a git external project in cmake, it automatically updates submodules. In the case of the previous PR, this was the case. One of GRPC submodule happens to be boringSSL :)

This is probably what will end up happening in this PR when no GRPC path is specified.